### ISSUE TITLE
cannot use Cthulhu with Julia 1.11.1

### ISSUE THREAD

**Issue by matthias314** [JuliaDebug/Cthulhu.jl#608]
```
~$ julia --startup=no
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.11.1 (2024-10-16)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.11) pkg> activate --temp
  Activating new project at `/tmp/jl_PF18di`

(jl_PF18di) pkg> add Cthulhu
   Resolving package versions...
    Updating `/tmp/jl_PF18di/Project.toml`
  [f68482b8] + Cthulhu v2.15.2
    Updating `/tmp/jl_PF18di/Manifest.toml`
  [1520ce14] + AbstractTrees v0.4.5
  [da1fd8a2] + CodeTracking v1.3.6
  [f68482b8] + Cthulhu v2.15.2
  [1eca21be] + FoldingTrees v1.2.1
  [70703baa] + JuliaSyntax v0.4.10
  [aea7be01] + PrecompileTools v1.2.1
  [21216c6a] + Preferences v1.4.3
  [d265eb64] + TypedSyntax v1.4.2
  [b8c1c048] + WidthLimitedIO v1.0.1
  [2a0f44e3] + Base64 v1.11.0
  [ade2ca70] + Dates v1.11.0
  [b77e0a4c] + InteractiveUtils v1.11.0
  [d6f4376e] + Markdown v1.11.0
  [de0858da] + Printf v1.11.0
  [3fa0cd96] + REPL v1.11.0
  [9a3f8284] + Random v1.11.0
  [ea8e919c] + SHA v0.7.0
  [6462fe0b] + Sockets v1.11.0
  [f489334b] + StyledStrings v1.11.0
  [fa267f1f] + TOML v1.0.3
  [cf7118a7] + UUIDs v1.11.0
  [4ec0a83e] + Unicode v1.11.0

julia> using Cthulhu

julia> @descend abs(1)
ERROR: MethodError: no method matching terminal(::REPL.LineEditREPL)
The function `terminal` exists, but no method is defined for this combination of argument types.

Closest candidates are:
  terminal(::REPL.LineEditREPL)
   @ REPL /usr/local/julia-1.11.1/share/julia/stdlib/v1.11/REPL/src/REPL.jl:588
  terminal(::REPL.LineEdit.PrefixSearchState)
   @ REPL /usr/local/julia-1.11.1/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:2097
  terminal(::REPL.LineEdit.SearchState)
   @ REPL /usr/local/julia-1.11.1/share/julia/stdlib/v1.11/REPL/src/LineEdit.jl:1983
  ...

Stacktrace:
 [1] default_terminal()
   @ Cthulhu /usr/local/julia-depot/packages/Cthulhu/GaSLZ/src/Cthulhu.jl:275
 [2] _descend_with_error_handling(f::Any, argtypes::Any; kwargs::@Kwargs{iswarn::Bool})
   @ Cthulhu /usr/local/julia-depot/packages/Cthulhu/GaSLZ/src/Cthulhu.jl:252
 [3] descend(::Any, ::Vararg{Any}; kwargs::@Kwargs{})
   @ Cthulhu /usr/local/julia-depot/packages/Cthulhu/GaSLZ/src/Cthulhu.jl:306
 [4] top-level scope
   @ REPL[4]:1
```
With Julia 1.10.5 it works.

**Comment by matthias314** [comment]
I think this is an instance of JuliaLang/julia#56216. I have set `JULIA_DEPOT_PATH`. Unsetting it makes the issue go away. So I believe this issue can be closed as it will probably be fixed in the next minor release of Julia 1.11.

**Comment by matthias314** [comment]
I've added a `:` at the end my `JULIA_DEPOT_PATH`. That fixed the problem for me.
